digi/xbee/exception.py

```diff
@@ -42,7 +42,14 @@ class ATCommandException(CommunicationException):
     All functionality of this class is the inherited of `Exception
     <https://docs.python.org/2/library/exceptions.html?highlight=exceptions.exception#exceptions.Exception>`_.
     """
-    pass
+    __DEFAULT_MESSAGE = "There was a problem sending the AT command packet."
+
+    def __init__(self, message=__DEFAULT_MESSAGE, cmd_status=None):
+        if not message:
```

Comment:
What is the reason for this? shouldn't the message be the default message if nothing is included as the message parameter or if what is included is None? Why the need to set it again?

Reply:
Removing it.